include/zproc.h

```diff
@@ -124,11 +124,6 @@ CZMQ_EXPORT int
 CZMQ_EXPORT void *
     zproc_actor (zproc_t *self);
 
-//  *** Draft method, for development use, may change without warning ***
```

Comment:
This one should stay - or rather be properly generated

Reply:
dumb question- shouldn't something like this be in ~platform.h ?

Reply:
yep - not sure why it's there

Reply:
@vyskocilm would defining this here: https://github.com/zeromq/zproject/blob/a7c75864064502de5fbc9fe2c996d03f7e4f0e20/zproject_vs20xx.gsl#L169

instead of include/zproc.h make more sense? that way it's also defined across all zmq projects from here too. can you test that to make sure it fits the problem you were solving?

(i haven't done this on windows- so bare with me, i don't wanna stomp on your fix by a zproject regen.. and i need to PR some changes to the zyre API which requires these to be somewhat in sync)